src/actions/index.js

```diff
@@ -1,9 +1,14 @@
 export const CreateBook = (obj) => ({
-  type: 'CreateBook',
+  type: 'CREATE_BOOK',
```

Comment:
![Screenshot from 2021-09-02 18-04-12](https://user-images.githubusercontent.com/46388113/131894957-6dfe1fb6-66bb-4afc-baa9-1e3b5bf56514.png)

Reply:
Kindly clone it properly the app is working just fine please refer the link below:
https://www.loom.com/share/4bd5c0db11bf482897c7e8f0ffe99e05